newton/solvers/mujoco/solver_mujoco.py

```diff
@@ -638,6 +638,75 @@ def update_dof_properties_kernel(
     dof_frictionloss[worldid, dof_in_env] = joint_friction[tid]
 
 
+@wp.kernel
+def update_geom_properties_kernel(
+    shape_collision_radius: wp.array(dtype=float),
+    shape_mu: wp.array(dtype=float),
+    shape_ke: wp.array(dtype=float),
+    shape_kd: wp.array(dtype=float),
+    shape_size: wp.array(dtype=wp.vec3f),
+    shape_transform: wp.array(dtype=wp.transform),
+    shape_to_geom: wp.array(dtype=wp.int32),
+    shapes_per_env: int,
+    up_axis: int,
+    # outputs
+    geom_rbound: wp.array2d(dtype=float),
+    geom_friction: wp.array2d(dtype=wp.vec3f),
+    geom_solref: wp.array2d(dtype=wp.vec2f),
+    geom_size: wp.array2d(dtype=wp.vec3f),
+    geom_pos: wp.array2d(dtype=wp.vec3f),
+    geom_quat: wp.array2d(dtype=wp.quatf),
+):
+    """Update geom properties from Newton shape properties."""
+    tid = wp.tid()
+    worldid = tid // shapes_per_env
+    shape_in_env = tid % shapes_per_env
+
+    geom_idx = shape_to_geom[shape_in_env]
+    if geom_idx < 0:  # Invalid mapping
+        return
+
+    # Update bounding radius
+    geom_rbound[worldid, geom_idx] = shape_collision_radius[tid]
+
+    # Update friction (slide, torsion, roll)
+    mu = shape_mu[tid]
+    geom_friction[worldid, geom_idx] = wp.vec3f(mu, 0.05 * mu, 0.05 * mu)  # Use scaled values for torsion/roll
+
+    # Update solref (stiffness, damping as time constants)
+    # MuJoCo uses time constants, Newton uses direct stiffness/damping
+    # Convert using heuristic: time_const = sqrt(mass/stiffness)
+    # For now, use a simplified mapping
+    ke = shape_ke[tid]
+    kd = shape_kd[tid]
+    if ke > 0.0:
+        # Approximate time constant based on typical contact scenarios
+        time_const_stiff = 0.02  # Default 20ms
```

Comment:
Can you make these constants part of the inputs to the `MuJoCoSolver`? Is this number supposed to equal the sim time step, in that case it could use the solver's time step?